src/sass/letters.scss

```diff
@@ -35,6 +37,15 @@
     padding-bottom: 1.5rem;
   }
 
+  .download-button {
+    margin: 1.5em 0 1em;
+
+    button:disabled {
```

Comment:
Maybe `> button:disabled`? I think it's good practice to make sure nested styles like this are as narrowly specified as possible.